src/sorcha/ephemeris/simulation_geometry.py

```diff
@@ -195,9 +195,19 @@ def barycentricObservatoryRates(et, obsCode, observatories, Rearth=RADIUS_EARTH_
     pos = posvel[0:3]
     vel = posvel[3:6]
     # Get the matrix that rotates from the Earth's equatorial body fixed frame to the J2000 equatorial frame.
-    m = spice.pxform("ITRF93", "J2000", et)
-    mp = spice.pxform("ITRF93", "J2000", et + delta_et)
-    mm = spice.pxform("ITRF93", "J2000", et - delta_et)
+    # We don't need to compute this repeatedly
+    et_1962 = spice.str2et("1962-Jan-20")
+    if et >= et_1962:
+        m = spice.pxform("ITRF93", "J2000", et)
+        mp = spice.pxform("ITRF93", "J2000", et + delta_et)
+        mm = spice.pxform("ITRF93", "J2000", et - delta_et)
+    else:
+        m = spice.pxform("IAU_EARTH", "J2000", et)
+        mp = spice.pxform("IAU_EARTH", "J2000", et + delta_et)
+        mm = spice.pxform("IAU_EARTH", "J2000", et - delta_et)
+    # m = spice.pxform("ITRF93", "J2000", et)
```

Comment:
JOSS review hasn't happened so any new additions will be reviewed - can we remove the commented out rows so we don't get asked to clean up during JOSS review?